docs/topics/dokkatoo-migration.md

```diff
@@ -0,0 +1,359 @@
+[//]: # (title: Migrate to Dokka Gradle plugin v2)
+
+> The Dokka Gradle plugin v2 is an [experimental](https://kotlinlang.org/docs/components-stability.html#stability-levels-explained) feature. 
+> It may be changed at any time. We appreciate your feedback in [GitHub](https://github.com/Kotlin/dokka/issues).
```

Comment:
*on* GitHub? [Source](https://ell.stackexchange.com/questions/89101/in-on-or-at-github)

Reply:
Thanks for sharing the explanation source.